### PR TITLE
Rename Model Load/Unload from Control to ModelControl

### PR DESCRIPTION
**Pull request by CoderHam**

### CODE REVIEW

File: src/clients/c++/request.h [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018-2019, NVIDIA CORPORATION. All rights reserved.
+ // Copyright (c) 2018-2019, NVIDIA CORPORATION. All rights reserved.
```
Comment (GuanLuo) [inline src/clients/c++/request.h]: space

File: src/core/server_status.proto [hunk 2]
```diff
@@ -350,9 +350,9 @@ message ServerStatus
...
   HealthRequestStats health_stats = 8;
 
-  //@@  .. cpp:var:: ControlRequestStats control_stats
+  //@@  .. cpp:var:: ModelControlRequestStats control_stats
   //@@
-  //@@     Statistics for Control requests.
+  //@@     Statistics for ModelControl requests.
   //@@
-  ControlRequestStats control_stats = 9;
+  ModelControlRequestStats control_stats = 9;
```
Comment (GuanLuo) [inline src/core/server_status.proto]: Also rename `control_stats`? Is your new API going to add similar thing here?

Comment (CoderHam) [reply src/core/server_status.proto]: No, I don't think I will need to add Stats for Register/Unregistering Shared Memory Regions.

Comment (deadeyegoodwin) [reply src/core/server_status.proto]: We should have stats for all the APIs. There is no reason to treat register/unregister differently. So control_stats needs to change to model_control_stats.

Comment (CoderHam) [reply src/core/server_status.proto]: Already done. Will incorporate the stats functionality in my changes for the shared memory control stats

**Pull request merged**